data/json/mutations/mutations.json

```diff
@@ -1136,12 +1136,12 @@
     "type": "mutation",
     "id": "XXXL",
```

Comment:
How will this interact with starting mutations being impossible to remove?  If your character loses all the weight they'll still be permanently unable to gain the canceled traits, is that correct?   I'm wondering if this might be a good exception to have in the system?  Because if I'm understanding correctly all the rest of the penalties are weight related and will go away as your character loses weight.

Reply:
XXXL weight trait is removed right after the game start. So nothing will prevent players to get previously-canceled traits during the game.

Reply:
Well that’s neat!